cvat/apps/engine/views.py

```diff
@@ -210,15 +216,11 @@ def get_serializer(self, *args, **kwargs):
     def about(request):
         from cvat import __version__ as cvat_version
         about = {
-            "name": "Computer Vision Annotation Tool",
+            "name": settings.ABOUT_INFO["name"],
+            "title": settings.ABOUT_INFO["title"],
+            "description": settings.ABOUT_INFO["description"],
```

Comment:
Why did you make the name and description customizable? I don't think that was in the customer's request. It's also not mentioned in the changelog.

Reply:
For me it makes sense if we want to customize the `/about` response via settings we can also customize those fields.

Reply:
I don't think we should make things customizable without a clear use case.

Reply:
Ok, removed. Lets keep only subtitle for now